_includes/content-toggle.liquid

```diff
@@ -21,85 +20,12 @@
         {% assign togglePath=thisContent[2] %}
-        {% capture toggleText %}{% include {{togglePath}} %}{% endcapture %}
         <div id="{{toggleName}}" class="panel-collapse collapse">
             <div class="panel-body">
-                {{ toggleText | markdownify }}
+                {% capture toggleText %}{% include {{togglePath}} %}{% endcapture %}
+                <table><tr><td>{{ toggleText | markdownify }}</td></tr></table>
             </div>
         </div>
         {% endfor %}
     </div>
 </div>
-
-<script>
-    jqueryUiDefer(initContentToggle{{contentToggleId}});
-    let toggleBlocksIdsToTocsIds{{contentToggleId}} = {};
-    function initContentToggle{{contentToggleId}}() {
-        $(function() {
-            if ($('#table-of-contents').find('#markdown-toc').length) {
-                let toggleBlocksIds = [];
-                $('.panel-heading.{{contentToggleId}} > a').each(function() {
-                    toggleBlocksIds.push($(this).attr("data-target").substring(1));
-                });
-                let i = 0;
-                for (let id of toggleBlocksIds) {
-                    toggleBlocksIdsToTocsIds{{contentToggleId}}[id] = [];
-                    $('#' + id).find(":header").each(function() {
-                        let heading = $(this);
-                        heading.attr("id", heading.attr("id") + i++);
-                        toggleBlocksIdsToTocsIds{{contentToggleId}}[id].push('markdown-toc-' + $(this).attr('id'));
-                    });
-                }
-                window.addEventListener('popstate', onPopState{{contentToggleId}});
-
-                onPopState{{contentToggleId}}();
-
-                $('.tb-liquid-content-toggle#{{contentToggleId}} a.content-toggle-button').click(function(event) {
-                    event.preventDefault();
-                    var id = $(this).attr("data-target").substring(1);
-                    var param = '{{contentToggleId}}';
-                    var params = Qs.parse(window.location.search, { ignoreQueryPrefix: true });
-                    params[param] = id;
-
-                    var newurl = window.location.protocol + "//" + window.location.host + window.location.pathname + '?' + Qs.stringify(params);
-                    if (window.location.hash) {
-                        newurl += window.location.hash;
-                    }
-                    window.history.pushState({ path: newurl }, '', newurl);
-                    selectTarget{{contentToggleId}}(id);
-                });
-            } else {
-                setTimeout(function() {
-                    initContentToggle{{contentToggleId}}();
-                }, 50);
-            }
-        });
-    }
-
-    function onPopState{{contentToggleId}}() {
-        var params = Qs.parse(window.location.search, { ignoreQueryPrefix: true });
-        var targetId = params['{{contentToggleId}}'];
-        if (!targetId) {
-            targetId = '{{firstTargetId}}';
```

Comment:
Where is variable "firstTargetId" initialized? We may need to delete unnecessary variables.